lib/indocker/repositories/clonner.rb

```diff
@@ -9,9 +9,16 @@ def initialize(configuration, logger)
   def clone(session, repository)
     raise ArgumenError.new("only git repositories should be clonned") if !repository.is_git?
 
-    session.exec!("rm -rf #{repository.clone_path} && mkdir -p #{repository.clone_path}")
-    
-    git_command = "git clone -b #{repository.branch} --depth 1 #{repository.remote_url} #{repository.clone_path}"
+    repository_remote_url = session.exec!("mkdir -p #{repository.clone_path} && cd #{repository.clone_path} && git config --get remote.origin.url").stdout_data.chomp
```

Comment:
1. you perform 2 different actions here, create a directory (which has a side effect) and get current remote url. I would separate them to 2 actions, because they do different tasks.
2. I think it would be better to create a separate private method "get_current_remote_url", so your intention will be clear.

Reply:
👍